app/code/Magento/StoreGraphQl/etc/schema.graphqls

```diff
@@ -31,4 +31,5 @@ type StoreConfig @doc(description: "The type contains information about a store
     secure_base_static_url : String @doc(description: "Secure base static URL for the store")
     secure_base_media_url : String @doc(description: "Secure base media URL for the store")
     store_name : String @doc(description: "Name of the store")
+    stores: [StoreConfig] @doc(description: "Contains information about all stores")
```

Comment:
Schema provides possibility to create a recursion with field 'stores' - but value will never be populated.
Do we really want to leave it as it is?

E.g. Query:
```
{
  storeConfig {
    id
    code
    website_id
    locale
    base_currency_code
    default_display_currency_code
    timezone
    weight_unit
    base_url
    base_link_url
    base_static_url
    base_media_url
    secure_base_url
    secure_base_link_url
    secure_base_static_url
    secure_base_media_url
    store_name
    stores {
      id
      code
      website_id
      locale
      base_currency_code
      default_display_currency_code
      timezone
      weight_unit
      base_url
      base_link_url
      base_static_url
      base_media_url
      secure_base_url
      secure_base_link_url
      secure_base_static_url
      secure_base_media_url
      store_name
      stores {
        id
        code
        website_id
        locale
        base_currency_code
        default_display_currency_code
        timezone
        weight_unit
        base_url
        base_link_url
        base_static_url
        base_media_url
        secure_base_url
        secure_base_link_url
        secure_base_static_url
        secure_base_media_url
        store_name
        stores {
          id
        }
      }
    }
  }
}
```

Response:
```
{
  "data": {
    "storeConfig": {
      "id": 1,
      "code": "default",
      "website_id": 1,
      "locale": "en_US",
      "base_currency_code": "USD",
      "default_display_currency_code": "USD",
      "timezone": "UTC",
      "weight_unit": "lbs",
      "base_url": "http://magento2/",
      "base_link_url": "http://magento2/default/",
      "base_static_url": "http://magento2/pub/static/version1590678667/",
      "base_media_url": "http://magento2/pub/media/",
      "secure_base_url": "https://magento2/",
      "secure_base_link_url": "https://magento2/default/",
      "secure_base_static_url": "https://magento2/pub/static/version1590678667/",
      "secure_base_media_url": "https://magento2/pub/media/",
      "store_name": null,
      "stores": [
        {
          "id": 1,
          "code": "default",
          "website_id": 1,
          "locale": "en_US",
          "base_currency_code": "USD",
          "default_display_currency_code": "USD",
          "timezone": "UTC",
          "weight_unit": "lbs",
          "base_url": "http://magento2/",
          "base_link_url": "http://magento2/default/",
          "base_static_url": "http://magento2/pub/static/version1590678667/",
          "base_media_url": "http://magento2/pub/media/",
          "secure_base_url": "https://magento2/",
          "secure_base_link_url": "https://magento2/default/",
          "secure_base_static_url": "https://magento2/pub/static/version1590678667/",
          "secure_base_media_url": "https://magento2/pub/media/",
          "store_name": null,
          "stores": null
        },
        {
          "id": 2,
          "code": "new_store_view",
          "website_id": 1,
          "locale": "en_US",
          "base_currency_code": "USD",
          "default_display_currency_code": "USD",
          "timezone": "UTC",
          "weight_unit": "lbs",
          "base_url": "http://magento2/",
          "base_link_url": "http://magento2/new_store_view/",
          "base_static_url": "http://magento2/pub/static/version1590678667/",
          "base_media_url": "http://magento2/pub/media/",
          "secure_base_url": "https://magento2/",
          "secure_base_link_url": "https://magento2/new_store_view/",
          "secure_base_static_url": "https://magento2/pub/static/version1590678667/",
          "secure_base_media_url": "https://magento2/pub/media/",
          "store_name": null,
          "stores": null
        }
      ]
    }
  }
}
```